Changing the steering committee (#444)
crystal-lang/crystal-website: _posts/2023-05-15-update-in-steering-committee.md

---
title: "Changes in the steering committee"
author: bziliani
summary: Juan Wajnerman is stepping down from his position, with Beta Ziliani taking his place.
---

> Juan Wajnerman has been involved with Crystal since its very beginning, and has served as one of the three Steering Committee members (the other two being Ary Borenzweig and Brian Cardiff). However, he decided to move on with his personal projects and leave this position. I am honoured to be selected as Juan's replacement.

Reviewer: ```suggestion
Juan Wajnerman has been involved with Crystal since its very beginning, and has served as one of the three Steering Committee members (the other two being Ary Borenzweig and Brian Cardiff). However, he decided to leave this position and move on with his personal projects. I am honoured to be selected as Juan's replacement.
```

> We can consider ourselves lucky to have shared paths with Juan, a great colleague with fine technical qualities that make him stand out. Our doors will be forever open in case he decides to come back to share his knowledge with the community of the language he helped grow.

Reviewer: ```suggestion
We consider ourselves lucky to have shared paths with Juan, a great colleague with fine technical qualities that make him stand out. Our doors will forever be open in case he decides to return to share his knowledge with the community for the language he helped grow.
```

> Juan Wajnerman has been involved with Crystal since its very beginning, and has served as one of the three Steering Committee members (the other two being Ary Borenzweig and Brian Cardiff). However, he decided to leave this position and move on with his personal projects. Beta Ziliani, who's been leading the development of the language for the past two years, will now take his seat.
> 
> We consider ourselves lucky to have shared paths with Juan, a great colleague with fine technical qualities that make him stand out. Our doors will forever be open in case he decides to return to share his knowledge with the community for the language he helped grow.

Reviewer 2: I would restructure the beginning to start with an introduction of the Steering Committee, then going into the current story about the transition Juan -> Beta.
Also I'd pull up the retrospection about Juan's contribution and include a thank you.
The replacement info should follow in a separate paragraph and maybe include some more details about Beta...

Rough suggestion:
```suggestion
The [Steering Council](https://crystal-lang.org/community/governance/#the-steering-council) is Crystal's core government authority. It's formed of the original creators, Ary Borenzweig, Brian Cardiff and Juan Wajnerman. They have been involved in the development of the language since its very beginning and also serve as Core Team members.

Juan has decided to leave his positions in the community and move on with his personal projects. 
We consider ourselves lucky to have shared paths with Juan, a great colleague with fine technical qualities that make him stand out.
Thank you, Juan for the great work you put into Crystal. Our doors will forever be open to share your knowledge with the community of the language you helped to grow.

The seat on the council goes to Beta Ziliani. He has been leading the development of the language for the past two years. With a background in programming language research and his proven skills in leadership, we are happy to give the position into capable hands.
```
